src/plan/aggregate.rs

```diff
@@ -28,14 +28,20 @@ pub enum AggregationFn {
     SUM,
     /// Average
     AVG,
-    ///Varianve
-    VARIANCE,
+    // ///Variance
+    // VARIANCE,
     // /// Standard deviation
     // STDDEV,
 }
 
+fn reorder(result_vec: Vec<Value>, positions: Vec<usize>) -> Vec<Value> {
+    let mut v: Vec<_> = result_vec.iter().zip(positions.iter()).collect();
+    v.sort_by(|a, b| a.1.cmp(b.1));
+    v.iter().map(|x| x.0.clone()).collect()
+}
+
 /// [WIP]
-/// A plan stage applying the specified aggregation function to
+/// A plan stage applying the specified single aggregation function to
```

Comment:
Some explanation of the differences to AggregateMultiple and why we want to have both would be helpful here.